hack/lib/protoc.sh

```diff
@@ -66,6 +69,7 @@ function kube::protoc::protoc() {
     PATH="${gogopath}:${PATH}" protoc \
       --proto_path="$(pwd -P)" \
       --proto_path="${KUBE_ROOT}/vendor" \
+      --proto_path="${KUBE_ROOT}/third_party/protobuf" \
```

Comment:
where is this file coming?

Reply:
or folder?

Reply:
this matches the import path passed in update-generated-protobuf-dockerized.sh:

```
PATH="${KUBE_ROOT}/_output/bin:${PATH}" \
  "${gotoprotobuf}" \
  --proto-import="${KUBE_ROOT}/vendor" \
  --proto-import="${KUBE_ROOT}/third_party/protobuf" \
  --packages="$(IFS=, ; echo "$*")" \
  --go-header-file "${KUBE_ROOT}/hack/boilerplate/boilerplate.generatego.txt"
```

so protoc can find `google/protobuf/descriptor.proto` consistently